wfl/Database/MongoDBManager.cs

```diff
@@ -26,6 +27,16 @@ public MongoDBManager()
             await collection.InsertOneAsync(r);
         }
 
+        public async Task<RestaurantDto> UpdateRestaurantNameAsync(string id, string name)
```

Comment:
Sounds like a plan

- [ ] Refactor logic to update all fields from the `RestarauntDto`